src/app/pages/patientsList/patientsList.component.html

```diff
@@ -20,6 +21,9 @@
                 <td>{{ patient?.firstName }}</td>
                 <td>{{ patient?.lastName }}</td>
                 <td> {{ patient?.birthdate }}</td>
+                <td><button [routerLink]="['/pages/doctorsList/patient/', patient.id]"
```

Comment:
Ten przycisk powinien być widoczny tylko dla recepcjonistki. Czy powinna powstać jakaś `ROLE_RECEPTIONIST`?

Reply:
powinna ;) nie ma takiej narazie xd mozesz dac adminowi tylko na razie

Reply:
Powinna, ale u nas recepjonistka to prawie admin xD